controllers/cluster/capabilities.go

```diff
@@ -0,0 +1,42 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	v1 "github.com/openshift/api/config/v1"
+	clusterversion "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
```

Comment:
I'n not sure if `clusterversion` is the best name in that context.
I could be misunderstanding the context but I think either `configv1` (keeping to standards) or `openshiftconfig` might be better choices.

Reply:
I agree :+1: